pages/linux/nmcli.md

```diff
@@ -22,6 +22,14 @@
 
 `nmcli connection down uuid {{uuid}}`
 
-- Print statuses of network interfaces:
+- Create a static IPv4-only connection:
 
-`nmcli device status`
+`nmcli c add con-name enp1s0 ifname enp1s0 type ethernet ip4 10.0.0.7/8 gw4 10.0.0.1 ipv4.dns 10.0.0.1 ipv6.method ignore`
```

Comment:
I'm pretty sure there's a long form of the `c` here. Even though it makes the line even longer, perhaps we could use it here please?
```suggestion
`nmcli connection add con-name enp1s0 ifname enp1s0 type ethernet ip4 10.0.0.7/8 gw4 10.0.0.1 ipv4.dns 10.0.0.1 ipv6.method ignore`
```

Reply:
I wanted to avoid putting more options as it's long enough. Ignore simply does not configure IPv6.

Reply:
Ah, I see. Considering IPv6 is the future version of the Internet Protocol, I'm reluctant to go ahead with the `ipv6.method {{ignore}}` bit here unless there's a convincing and common use-case.

This said, it looks like you've made some pretty extensive changes since I last reviewed, so I'll do a fresh review.